securedrop/requirements/python3/securedrop-app-code-requirements.txt

```diff
@@ -108,33 +108,36 @@ cryptography==3.4.7 \
 flask-assets==0.12 \
     --hash=sha256:6031527b89fb3509d1581d932affa5a79dd348cfffb58d0aef99a43461d47847
     # via -r requirements/python3/securedrop-app-code-requirements.in
-flask-babel==0.11.2 \
-    --hash=sha256:462a3c599b0ccf426ca1757cc612f1db383844efd346d14170da04c8c76dd521 \
-    --hash=sha256:c0d75710bd4b0fe866f9f2347de6e19208712f9cec006436b4c1c15d4cb0c939
+flask-babel==2.0.0 \
+    --hash=sha256:e6820a052a8d344e178cdd36dd4bb8aea09b4bda3d5f9fa9f008df2c7f2f5468 \
+    --hash=sha256:f9faf45cdb2e1a32ea2ec14403587d4295108f35017a7821a2b1acb8cfd9257d
     # via -r requirements/python3/securedrop-app-code-requirements.in
```

Comment:
Diff reviewed in <https://github.com/freedomofpress/securedrop-debian-packaging/wiki/Flask-Babel-0.11.2-to-2.0.0>.